python/graphstorm/model/graph_utils.py

```diff
@@ -0,0 +1,208 @@
+"""
```

Comment:
do you make any change in this file? or you just moved the code

Reply:
The changes are L111-115 and L200-L204